assets/js/components/KeyMetrics/MetricsSelectionPanel/KeyMetricsError.js

```diff
@@ -77,9 +73,7 @@ function KeyMetricsError( { savedMetrics } ) {
 	);
 
 	const selectedMetricsCount = selectedMetrics?.length || 0;
-	const maxSelectedMetricsLimit = isConversionReportingEnabled
-		? MAX_SELECTED_METRICS_COUNT_WITH_CONVERSION_EVENTS
-		: MAX_SELECTED_METRICS_COUNT;
+	const maxSelectedMetricsLimit = MAX_SELECTED_METRICS_COUNT;
```

Comment:
Same here, we don't need this alias variable, let's use `MAX_SELECTED_METRICS_COUNT` directly.

Reply:
@eugene-manuilov this has been resolved in my latest commits.